public/lessons/postgres.md

```diff
@@ -12,75 +21,75 @@ To connect to your app to Postgres, you need a module called `pg`. You need to s
 ```
 const { Client } = require('pg')
 
-const startApp = async ()=>{
-    client = new Client({
-            host: '<host name>',
-                port: 5432,
-                    user: '<username>',
-                        password: '<password>',
-                            database: '<database_name>'
-                              })
-      await client.connect()
-          console.log('Connected')
+const startApp = async () => {
+  client = new Client({
+    host: '165.227.55.105',
+    port: 5432,
+    user: '@email',
+    password: '@dbPassword',
+    database: 'db_example'
+  })
+  await client.connect()
+  console.log('Connected')
 }
 
 startApp()
-  ```
+```
 
 ## 2.2 Simple Commands
   After connecting to Postgres, you can now send commands that Postgres can understand. These commands are called Sequel Query Language (SQL). Here are a few:
```

Comment:
I believe it should be Structured Query Language instead of Sequel Query Language